src/main/kotlin/net/atos/zac/app/exception/RestExceptionMapper.kt

```diff
@@ -59,7 +59,19 @@ class RestExceptionMapper : ExceptionMapper<Exception> {
     }
 
     /**
-     * Converts an exception to a JAX-RS response depending on the exception type.
+     * Converts an exception to a JAX-RS response based on the exception type.
```

Comment:
Nice one! Nice that you managed to find this exception mapper class too. Not easy to discover. ;-)